ssl/src/main/java/org/wildfly/security/ssl/SNISSLExplorer.java

```diff
@@ -341,10 +340,7 @@ public static int getRequiredSize(byte[] source,
      *     };
      * } ClientHello;
      */
-    private static List<SNIServerName> exploreClientHello(
-            ByteBuffer input,
-            byte recordMajorVersion,
-            byte recordMinorVersion) throws SSLException {
+    private static List<SNIServerName> exploreClientHello(ByteBuffer input) throws SSLException {
```

Comment:
Just a minor comment, but I think we could get away without remvoing these params. These could be used later on.

Reply:
@fjuma thanks for your feedback. I have removed this change from the PR. Please check again.